file_parse_test.go

```diff
@@ -0,0 +1,34 @@
+package samosa
+
+import (
+	"testing"
+)
+
+func Test_parse(t *testing.T) {
+	testOut := make(map[string]int)
+	err := readCoverReport("./coverage.out", testOut)
```

Comment:
Oh, missed this! We would need to store a test coverage report in a directory called `testdata`. [Example](https://github.com/burntcarrot/samosa/blob/main/internal/testdata/test_coverage.out)

Reply:
updated in commit https://github.com/burntcarrot/samosa/pull/4/commits/3c915a4a20b36852fc8812ac8508012d78d766ec

Reply:
@ehrktia The tests fail because the testdata is not present in this PR, could you please add a coverage report in `testdata/coverage.out` with the content being like [this](https://github.com/burntcarrot/samosa/blob/main/internal/testdata/test_coverage.out)?

Reply:
@burntcarrot done https://github.com/burntcarrot/samosa/pull/4/commits/1581df96b4b282220ec55db8174bd1e846bc6d32

Reply:
Thanks! Resolving this.

Reply:
added coverage in https://github.com/burntcarrot/samosa/pull/4/commits/1581df96b4b282220ec55db8174bd1e846bc6d32 commit